### PR TITLE
Updated TeamsJS CC Matrix App

### PR DESCRIPTION
**Pull request by vikramtha**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

> I added a revised version of the TeamsJS CC Matrix app which has addressed al the comments in the previous PR and included more support. I believe this needs to be on the TeamsJS SDK so that the samples that are linked in the public capability matrix point to the correct repo.

> If this Pull Request should close/resolve any issues when merged, use [the special syntax for that](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) here.

### Main changes in the PR:

1. Added the Teams JS CC App

### Validation performed:

None

### Unit Tests added:

Not necessary, sample app

### End-to-end tests added:

No
### Change file added:

> Ensure the change file meets the [formatting requirements](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball).

Not needed as this doesn't impact client facing code

### CODE REVIEW

File: apps/teamsjs-cc-app/readme.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# TeamsJS Capability Checker
```
Comment (AE-MS) [inline apps/teamsjs-cc-app/readme.md:1]: Can this file have some info about what this app is and what it's for?

Some of the info about how to run it is good and helpful and should stay -- but some details specific to this app would be good.

File: apps/teamsjs-cc-app/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noFallthroughCasesInSwitch": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "react-jsx"
```
Comment (AE-MS) [inline apps/teamsjs-cc-app/tsconfig.json:17]: Consider alphabetizing these

File: apps/teamsjs-cc-app/appPackage/manifest.json [hunk 3]
```diff
@@ -0,0 +1,121 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+    "manifestVersion": "1.16",
+    "version": "1.0.2",
+    "id": "${{TEAMS_APP_ID}}",
+    "packageName": "com.microsoft.teams.extension",
```
Comment (AE-MS) [inline apps/teamsjs-cc-app/appPackage/manifest.json:6]: >com.microsoft.teams.extension [](http://example.com/codeflow?start=20&length=29)

Is there a more specific name you can use for this?

File: apps/teamsjs-cc-app/appPackage/manifest.json [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    },
+    "permissions": [
+        "identity",
+        "messageTeamMembers"
+    ],
+    "devicePermissions": [
+        "geolocation",
+        "media",
+        "notifications",
+        "midi",
```
Comment (AE-MS) [inline apps/teamsjs-cc-app/appPackage/manifest.json:111]: >midi [](http://example.com/codeflow?start=9&length=4)

I hope you make good use of this. 😂

**Comment by AE-MS** [comment]
For future reference, putting screenshots of the app into the PR request can be helpful to reviewers.

**Comment by AE-MS** [comment]
I'll review more of the code once the tests are passing and the code has been updated to latest main.

**Comment by TrevorJoelHarris** [comment]
Talked with @vikramtha and he doesn't have time right now to prioritize finishing this change, so will close this PR for now. Feel free to reopen if you want to start preparing for checkin again.

**Pull request closed without merging**